packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -72,6 +74,10 @@ $ms-Persona-presenceSizeXl: 28px;
   display: flex;
   align-items: center;
 
+  &:focus {
+    outline: 1px solid $focusedBorderColor;
```

Comment:
I also see that @mikewheaton added `@mixin focus-outline` to `Link`, so if that's what you need, we should move it out of `Link` and maybe into common.scss.